File: iniciante/python/1154-idades.py
def idades():
    soma = 0
    quantidade_de_idades = 0
    while True:
        idade = int(input())
        if idade < 0:
            break
        soma += idade
        quantidade_de_idades += 1
    media = soma / quantidade_de_idades
    print(f'{media:.2f}')


idades()

File: iniciante/c/1154-idades.c
#include <stdio.h>
#include <stdbool.h>

int main(){
    int count = 0, idade;
    float idades_totais = 0, media;
    while (true){
        scanf("%d", &idade);
        if (idade < 0){
            break;
        }
        idades_totais += idade;
        count++;
    }
    media = idades_totais / count;
    printf("%.2f\n", media);

    return 0;
}
